<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HuacaoManagementClass</class>
 <widget class="QMainWindow" name="HuacaoManagementClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>滑槽管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="topImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1300</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="HuacaoManagement.qrc">:/HuacaoManagement/Resources/topBar.PNG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChannel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>滑槽组</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFlight">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>飞机型号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPolicy">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>分配策略</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetting">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>参数设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>110</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>航班</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>110</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>滑槽</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>110</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(158, 158, 158);
background-color: rgb(240, 240, 240);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="HuacaoManagement.qrc">
      <normaloff>:/HuacaoManagement/Resources/search.png</normaloff>:/HuacaoManagement/Resources/search.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="popupMode">
     <enum>QToolButton::DelayedPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>110</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>110</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>结束时间</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入航班号</string>
    </property>
    <property name="placeholderText">
     <string>请输入查询的航班号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>110</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>110</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>110</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="btnLock">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>110</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(158, 158, 158);
background-color: rgb(240, 240, 240);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="HuacaoManagement.qrc">
      <normaloff>:/HuacaoManagement/Resources/lock.png</normaloff>:/HuacaoManagement/Resources/lock.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>25</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="popupMode">
     <enum>QToolButton::DelayedPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>1240</width>
      <height>680</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>156</width>
      <height>36</height>
     </size>
    </property>
    <widget class="PaintTimeBar" name="tabEast">
     <attribute name="icon">
      <iconset resource="HuacaoManagement.qrc">
       <normaloff>:/HuacaoManagement/Resources/MU.jpg</normaloff>:/HuacaoManagement/Resources/MU.jpg</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabSouth">
     <attribute name="icon">
      <iconset resource="HuacaoManagement.qrc">
       <normaloff>:/HuacaoManagement/Resources/CZ.jpg</normaloff>:/HuacaoManagement/Resources/CZ.jpg</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PaintTimeBar</class>
   <extends>QWidget</extends>
   <header>PaintTimeBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="HuacaoManagement.qrc"/>
 </resources>
 <connections/>
</ui>
